store/eigenda.go

```diff
@@ -13,10 +13,12 @@ import (
 )
 
 type EigenDAStoreConfig struct {
-	MaxBlobSizeBytes     uint64
+	MaxBlobSizeBytes uint64
+	// the # of Ethereum blocks to wait after the L1BlockReference # before attempting to verify
+	// & accredit a blob
 	EthConfirmationDepth uint64
 
-	// The total amount of time that the client will spend waiting for EigenDA to confirm a blob
+	// total duration time that client waits for blob to confirm
```

Comment:
is StatusQueryTimeout related to EthConfirmationDepth above? If so, maybe a more informative variable name would help, to show they are related? I remember being confused by this variable first time I read the proxy code, and still am right now.

Reply:
they are separate - `StatusQueryTimeout` just refers to how long to wait for a blob to confirm before timing out client side

Reply:
so why not call it `ConfirmationTimeout` then?